index.html

```diff
@@ -0,0 +1,1255 @@
+                  SPONSORED INSIGHT
+                </div>
+              </figure>
+
+              <div class="card-body">
+                <a href="#" class="text-decoration-none text-dark"><h3 class="card-title fw-bold">Hokkaido: Japan’s Northern Frontier</h3></a>
+                <p class="card-text">Featuring a unique history, uninhibited wildness and at times otherworldly natural spectacles, Hokkaido offers an experience that is completely different from the rest of Japan.</p>
+              </div>
+            </div>
+
+            <div class="sponsors">
+              <div class="sponsor card mb-4">
+                <figure class="card-img-top position-relative">
+                  <a href="#"><img src="https://web.archive.org/web/20210120125445if_/https://d.newsweek.com/en/full/1701186/ph3095-01.jpg?w=500&h=333&f=b8162daf93e087918df15d58a79bfc35" alt="Japanese marsh" class="img-fluid"></a>
+                  <div class="position-absolute bottom-0 start-0 bg-light px-3 py-1 text-primary fw-bold fs-6">
+                    SPONSORED INSIGHT
+                  </div>
+                </figure>
+
+                <div class="card-body">
+                  <a href="#" class="text-decoration-none text-dark"><h3 class="card-title fw-bold">Hokkaido: Japan’s Northern Frontier</h3></a>
+                  <p class="card-text">Featuring a unique history, uninhibited wildness and at times otherworldly natural spectacles, Hokkaido offers an experience that is completely different from the rest of Japan.</p>
+                </div>
+              </div>
+
+              <div class="sponsor card mb-4">
+                <figure class="card-img-top position-relative">
+                  <a href="#"><img src="https://web.archive.org/web/20210120125445if_/https://d.newsweek.com/en/full/1680316/americas-most-responsible-companies-2021.png?w=500&h=333&f=ae10b886f06d2f293296bc4d2f90dda0" alt="" class="img-fluid"></a>
+                  <div class="position-absolute bottom-0 start-0 bg-light px-3 py-1 text-primary fw-bold fs-6">
+                    RANKINGS
+                  </div>
+                </figure>
+
+                <div class="card-body">
+                  <a href="#" class="text-decoration-none text-dark"><h3 class="card-title fw-bold">America's Most Responsible Companies 2021</h3></a>
+                  <p class="card-text">We partnered with Statista to rank the country's Most Responsible Companies. They share the values that we all prioritize, and that's worth recognizing in 2021</p>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div class="bottom-section row px-lg-3">
+        <div class="magazine">
+          <div class="bottom-section-title fw-bold text-uppercased-flex pb-2 justify-content-center align-items-center">
+            <span class="title-icon"></span>In the Magazine
```

Comment:
- [ ] Starting from the "In the Magazine" section it seems you forgot to make sure that the title of each section is made using an hx tag(h2, h3, h4, etc...).

Please check if all the section titles are made with an hx tag and fix the ones that are not.

Reply:
- [ ] Please fix the styling of the "In the Magazine" title:


**Yours**
![Screenshot_2021-01-29 Newsweek - News, Analysis, Politics, Business, Technology(1)](https://user-images.githubusercontent.com/16193212/106266523-4cefa900-6207-11eb-988d-6850d7e61476.png)
---
**Original**
![Screenshot_2021-01-29 Newsweek(3)](https://user-images.githubusercontent.com/16193212/106266543-524cf380-6207-11eb-8098-4fb39ff1435b.png)